### PR TITLE
{toolchain} intel/2023a

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3129.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/d97fcda35cc6a731eb6a41e6e6f876ae for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 5 out of 8 (4 easyconfigs in total)
interactive2 - Linux Rocky Linux 8.7, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/af9faa4599637df655368c7a439a5326 for a full test report.

**Comment by bedroge** [comment]
Ah, that failed because of https://github.com/easybuilders/easybuild-easyblocks/issues/2910. Let me try again with RPATH support disabled.

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 1 out of 4 (4 easyconfigs in total)
interactive2 - Linux Rocky Linux 8.7, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/1515cd2c8050d6be18308588cfffd91b for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
interactive2 - Linux Rocky Linux 8.7, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/0930a679231b494c097d43314b3d97ff for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Going in, thanks @boegel!

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 17 out of 17 (4 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.6
See https://gist.github.com/bedroge/92314c07deb686cdb0816fd4cd767aa1 for a full test report.
